### PR TITLE
Stylize Docs

### PR DESCRIPTION
**Pull request by sahariko**
## Main Change
Stylize auto-generated doc site:
- Add Passable colors.
- Make sidemenu more elegant.
- Add Passable logo to README.

### Additional Changes
- Minor text copy fixes.
- Fix `eslint` semi rule to be more specific.
- Reorder sidemenu by extracting "pass" from "getting started".

### CODE REVIEW

File: playground/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const passable = require('passable');
+const passable = require('../dist/Passable.min');
```
Comment (ealush) [inline playground/index.js]: Maybe even `../dev/Passable`, you'll need to have the dev server running, though

File: documentation/enforce/relationships/README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Enforce conditions relationships
 When running validations against your data, there are different relationships between your rules that you need to consider. The enforce function exposes three different functions that represent possible relationships.
 
-* `allOf`  | `AND` | makes sure all rules specified are true.
-* `anyOf`  | `OR`  | makes sure at least one rule is true.
-* `noneOf` | `NOT` | makes sure no rule is true.
+* `allOf` (AND) - makes sure all rules specified are true.
+* `anyOf` (OR) - makes sure at least one rule is true.
+* `noneOf` (NOT) - makes sure no rule is true.
```
Comment (ealush) [inline documentation/enforce/relationships/README.md]: I'd leave this change out. This area changed a lot after the recent `enforce` update.

File: documentation/README.md [hunk 3]
```diff
@@ -5,13 +5,13 @@ PASSABLE {{PASSABLE_VERSION}}
...
         * [Installation](./getting_started/installation.md)
         * [Writing tests](./getting_started/writing_tests.md)
-        * [The response object](./getting_started/response.md)
-        * [Passable params](./getting_started/params.md)
-        * [The pass function](./getting_started/pass/index.md)
-            * [How to fail a pass](./getting_started/pass/how_to_fail.md)
-            * [Warn only passes](./getting_started/pass/warn_only_passes.md)
-            * [Running a specific pass](./getting_started/pass/specific.md)
-    * [The Enforce Function](./enforce/README.md)
+        * [The "response" object](./getting_started/response.md)
```
Comment (ealush) [inline documentation/README.md]: I believe backticks for code blocks work in URLs too.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,4 @@
-PASSABLE
-========
+# ![Passable](https://raw.githubusercontent.com/fiverr/passable/master/documentation/assets/logo.svg "Passable")
```
Comment (ealush) [inline README.md]: 🥇

File: package.json [hunk 5]
```diff
@@ -40,6 +41,7 @@
     "babel-preset-flow": "^6.23.0",
     "chai": "^4.1.1",
     "cross-env": "^5.1.1",
+    "docpress": "^0.7.1",
```
Comment (ealush) [inline package.json]: nope. use npx. No need to add them as a dependency when it is all automated.

File: docpress.json [hunk 6]
```diff
@@ -1,5 +1,9 @@
...
   "docs": "documentation",
   "github": "fiverr/passable",
-  "dist": "docs"
-}
\ No newline at end of file
+  "dist": "docs",
+  "css": [
+      "https://unpkg.com/highlight.js@9.12.0/styles/atom-one-light.css",
+      "documentation/assets/style/main.css"
+  ]
```
Comment (ealush) [inline docpress.json]: while at it, what do you think about adding passable directly as a script tag to the page, so people can use the console to play with it?

Comment (sahariko) [reply docpress.json]: I'll look into that 👍

Comment (sahariko) [reply docpress.json]: `docpress` doesn't behave nicely when trying to add local scripts. Maybe we can use the unpkg version in a future improvement.

File: playground/index.js (hunk 1 above)
Comment (sahariko) [reply playground/index.js]: I'm not sure it's in this PR's scope, but sounds good.

File: documentation/enforce/README.md [hunk 7]
```diff
@@ -48,34 +48,41 @@ All the following are valid uses of enforce.
...
-    }).smallerThan(150);
+    largerThan: 5, // in anyOf - this one is enough to set the anyOf block to true
+    smallerThan: 6,
+}).noneOf({
+    isString: true, // in noneOf, all must be false
+    isNumber: true
+}).isArray(); // true
 ```
 
-Enforce exposes all [predefined](./rules/README.md) and [custom](./rules/custom.md) rules, and the [compound enforcement methods](./compound/README.md). You may use chaining to make multiple enfocements for the same value.
```
Comment (ealush) [inline documentation/enforce/README.md]: ?

File: documentation/enforce/relationships/README.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
+# Enforce conditions relationships
```
Comment (ealush) [inline documentation/enforce/relationships/README.md]: This file has been deleted.

File: documentation/README.md [hunk 9]
```diff
@@ -5,13 +5,14 @@ PASSABLE {{PASSABLE_VERSION}}
...
-            * [Running a specific pass](./getting_started/pass/specific.md)
-    * [The Enforce Function](./enforce/README.md)
+        * [The `response` object](./getting_started/response.md)
+        * [Passable Parameters](./getting_started/params.md)
+    * [The `pass` Function](./pass/index.md)
+        * [How to fail a pass](./pass/how_to_fail.md)
+        * [Warn only passes](./pass/warn_only_passes.md)
+        * [Running a specific pass](./pass/specific.md)
+    * [The `enforce` Function](./enforce/README.md)
+        * [Enforce relationships](./enforce/relationships/README.md)
```
Comment (ealush) [inline documentation/README.md]: This file has been deleted

**Pull request merged**